### PR TITLE
Toc: implement Debug trait

### PR DESCRIPTION
**Pull request by fengalin**

### CODE REVIEW

File: gstreamer/src/toc.rs [hunk 1]
```diff
@@ -99,6 +100,15 @@ impl ToOwned for TocRef {
     }
 }
 
+impl fmt::Debug for TocRef {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("Toc")
+            .field("scope", &self.get_scope())
+            .field("tags", &self.get_tags())
```
Comment (sdroege) [inline gstreamer/src/toc.rs]: You probably also want to dump the entries here?

File: gstreamer/src/toc.rs [hunk 2]
```diff
@@ -237,6 +247,20 @@ impl ToOwned for TocEntryRef {
...
+impl fmt::Debug for TocEntryRef {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("TocEntry")
+            .field("entry_type", &self.get_entry_type())
+            .field("uid", &self.get_uid())
+            .field("start_stop", &self.get_start_stop_times())
+            .field("tags", &self.get_tags())
+            .field("is_alternative", &self.is_alternative())
+            .field("is_sequence", &self.is_sequence())
+            .field("loop", &self.get_loop())
```
Comment (sdroege) [inline gstreamer/src/toc.rs]: Sub entries here?

Comment (fengalin) [reply gstreamer/src/toc.rs]: I don't know. In that case, this should be recursive. I though the user would choose how and when to traverse the tree.

Comment (sdroege) [reply gstreamer/src/toc.rs]: Debug impls usually recurse into all their children

Comment (fengalin) [reply gstreamer/src/toc.rs]: Ok then.

**Pull request merged**